src/Apps/Sell/Components/BottomFormNavigation.tsx

```diff
@@ -76,22 +76,24 @@ const BottomFormNextButton = () => {
   const { showAuthDialog } = useAuthDialog()
   const { trackTappedContinueSubmission } = useSubmissionTracking()
 
-  const {
-    state: { loading },
-  } = useSellFlowContext()
-
   const {
     actions,
-    state: { submissionID, isSubmitStep, nextStep },
+    state: { submissionID, isSubmitStep, nextStep, loading },
   } = useSellFlowContext()
 
   const onNext = async () => {
-    if (!isLoggedIn) {
+    if (!isLoggedIn && isSubmitStep) {
```

Comment:
I wonder whether we should track how many users reach the last step without being logged in 🤔